numpy/lib/npyio.py

```diff
@@ -475,9 +475,7 @@ def save(file, arr, allow_pickle=True, fix_imports=True):
 
     Notes
     -----
-    For a description of the ``.npy`` format, see the module docstring
-    of `numpy.lib.format` or the NumPy Enhancement Proposal
-    http://numpy.github.io/neps/npy-format.html
+    For a description of the ``.npy`` format, see :py:mod:`numpy.lib.format`
```

Comment:
Does this actually render as a link to the documentation?

Reply:
it did for me when I built locally, this was the only way I **could** get a link to render. I expected that I could have used `:ref:something` but I couldn't figure out the `something`